apps/allocations/app/components/App/App.js

```diff
@@ -182,6 +182,6 @@ class App extends React.PureComponent {
 }
 
 export default () => {
-  const { api, appState } = useAragonApi()
-  return <App api={api} {...appState} />
+  const { api, appState, displayMenuButton } = useAragonApi()
+  return <App api={api} {...appState} displayMenuButton={displayMenuButton} />
```

Comment:
It looks like we need this, but I'm not sure why it got included on this PR. I guess it was missed during the #994 effort? Thanks for spotting this!

Reply:
It was coming from a different place before 'api' was introduced. Now it needs to be here.